### PR TITLE
CDAP-17462 Add supportRowId to source properties and chagne eventsOrdered to boolean

### PR DESCRIPTION
**Pull request by seanzhougoogle**
CDAP-17462 Add supportRowId to source properties and chagne eventsOrdered to boolean

### CODE REVIEW

File: delta-api/src/main/java/io/cdap/delta/api/SourceProperties.java [hunk 1]
```diff
@@ -26,19 +26,15 @@
   /**
    * Enum describing ordering of events.
```
Comment (sagarkapare) [inline delta-api/src/main/java/io/cdap/delta/api/SourceProperties.java]: Remove this comment too?

**Review by sagarkapare — APPROVED** [review]

File: delta-api/src/main/java/io/cdap/delta/api/SourceProperties.java [hunk 2]
```diff
@@ -26,19 +26,15 @@
...
    */
-  public enum Ordering {
-    ORDERED,
-    UN_ORDERED
-  }
 
-  private final Ordering ordering;
+  private final boolean supportRowId;
```
Comment (sagarkapare) [inline delta-api/src/main/java/io/cdap/delta/api/SourceProperties.java]: nit: should we rename this to `rowIdSupported` and corresponding getter can be `isRowIdSupported` and setter `setRowIdSupported`?

File: delta-api/src/main/java/io/cdap/delta/api/SourceProperties.java (hunk 1 above)
Comment (seanzhougoogle) [reply delta-api/src/main/java/io/cdap/delta/api/SourceProperties.java]: done:)

File: delta-api/src/main/java/io/cdap/delta/api/SourceProperties.java (hunk 2 above)
Comment (seanzhougoogle) [reply delta-api/src/main/java/io/cdap/delta/api/SourceProperties.java]: done :)

**Pull request merged**